src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -104,12 +100,11 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
   }
 
   requiresUnifiedPlan(): boolean {
-    let shouldEnable =
-      this.isFirefox() || (this.hasWebKitWebRTC() && this.isUnifiedPlanSupported());
-    if (this.enableUnifiedPlanForChromiumBasedBrowsers) {
-      shouldEnable = shouldEnable || this.hasChromiumWebRTC();
-    }
-    return shouldEnable;
+    return (
+      this.isFirefox() ||
+      this.hasChromiumWebRTC() ||
+      (this.hasWebKitWebRTC() && this.isUnifiedPlanSupported())
```

Comment:
Question: Is there a Safari version that unified plan is not supported?

Reply:
Safari 12: https://github.com/aws/amazon-chime-sdk-js/pull/595